openedx/features/enterprise_support/admin/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+
+class CSVImportForm(forms.Form):
+    csv_file = forms.FileField(label='CSV File')
```

Comment:
I think we should also add validation logic here to validate the format of the CSV file.